scripts/nayduck.py

```diff
@@ -182,11 +182,16 @@ def run_locally(tests):
         # See nayduck specs at https://github.com/near/nayduck/blob/master/lib/testspec.py
         fields = test.split()
 
+        timeout = ""
         index = 1
+        ignored = []
         while len(fields) > index and fields[index].startswith('--'):
+            if fields[index].startswith("--timeout="):
+                timeout = fields[index].split("=")[1]
+            else:
+                ignored.append(fields[index])
             index += 1
 
-        ignored = fields[1:index]
         del fields[1:index]
         message = f'Running ‘{"".join(fields)}’'
```

Comment:
By the way, since you’re editing this file, could you fix this join to be `" ".join(fileds)`? Same on line 198.

Reply:
Ok, fixed.